src/components/ButtonPanel/ButtonPanel.js

```diff
@@ -1,46 +1,51 @@
 import React from 'react';
 import Button from '../Button/Button';
 
-const buttonPanel = () => {
+const buttonPanel = (props) => {
```

Comment:
## [OPTIONAL BUT STRONGLY RECOMMENDED CHANGE]
We only have one `prop` being passed from `App.js` so we can deconstruct this directly in the argument and will just do an implicit `return`:

```javascript
const buttonPanel = ({ clickHandler }) => (
 <div>
   .....
</div>
)
```

Or a more classic way but still requires the `return` statement:

```javascript
const buttonPanel = (props) => {
  const { clickHandler } = props;
}
return (
 <div>
   ....
</div>
)
```

Reply:
My comment from above won't let me implement this. :neutral_face:

Reply:
Rebuttal to the comment above made :)